### DISCUSSION TITLE
Docker guide not correct

### DISCUSSION THREAD

**Discussion by lokialice** [mixcore/mix.core#517] [Q&A]
From guide start with docker and dock-compose already failed
Please check that, the port mapping is wrong, in container run port 5000 but it's expose port 80
...

**Comment by Smilefounder** [comment]
thanks @lokialice  I haven't checked docker image since last year. Let me take a look.

**Reply by lokialice** [reply]
Thanks

**Comment by Smilefounder** [comment]
@lokialice as I checked, my ports mapping are correct. As once you deploy to the server, you can use port 80/443 by default instead of 5000/5001.

This is to reduce the effort to manually config the proxy by ourselves.

Please feel free to correct me if you have any other better ideas. As I'm not good at Docker configuration.
